<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toDo</class>
 <widget class="QMainWindow" name="toDo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>toDo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="tableView_2"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableView" name="tableView_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Files">
    <property name="title">
     <string>&amp;Files</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete_TAsk"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_Files"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Task"/>
   <addaction name="actionPending_Task"/>
   <addaction name="actionFinished_Task"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionNew_Task">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/new Task.png</normaloff>:/new Task.png</iconset>
   </property>
   <property name="text">
    <string>New Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPending_Task">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons8-data-pending-50.png</normaloff>:/icons8-data-pending-50.png</iconset>
   </property>
   <property name="text">
    <string>Pending Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFinished_Task">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons8-task-completed-50.png</normaloff>:/icons8-task-completed-50.png</iconset>
   </property>
   <property name="text">
    <string>Finished Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons8-edit-30.png</normaloff>:/icons8-edit-30.png</iconset>
   </property>
   <property name="text">
    <string> Edit Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDelete_TAsk">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons8-delete-80.png</normaloff>:/icons8-delete-80.png</iconset>
   </property>
   <property name="text">
    <string>Delete TAsk</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons8-exit-64.png</normaloff>:/icons8-exit-64.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>toDo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFinished_Task</sender>
   <signal>triggered(bool)</signal>
   <receiver>tableView_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPending_Task</sender>
   <signal>triggered(bool)</signal>
   <receiver>tableView_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
